lib/tabs/KTabsList.vue

```diff
@@ -63,7 +65,22 @@
         </slot>
       </button>
     </template>
-  </div>
+    <template #more="{ overflowItems }">
+      <KIconButton
+        tooltip="More"
```

Comment:
This should be a prop -- something like `moreButtonTooltip` because we don't have any i18n machinery setup in KDS.

So in Kolibri or Studio, we would need a translation we can pass into that prop like:

`<KTabsList :moreButtonTooltip="coreStrings('moreLabel')" /* ... */>`

That way we'd pass in the translated version of the string to be used as the tooltip label.